sys/console/full/syscfg.yml

```diff
@@ -26,6 +26,9 @@ syscfg.defs:
     CONSOLE_SEMIHOSTING:
         description: 'Set console output to ARM semihosting'
         value: 0
+    CONSOLE_USB:
```

Comment:
This can't be enabled if `CONSOLE_UART` is, it would be nice to have it described on `syscfg.restrictions`